src/group/components/GroupsHomeCard.js

```diff
@@ -18,34 +18,44 @@ import theme from 'theme';
 const GroupItem = ({ group }) => {
   const { t } = useTranslation();
   return (
-    <List>
+    <ListItem>
       <Link
         component={RouterLink}
         underline="none"
         to={`/groups/${group.slug}`}
       >
         {group.name}
       </Link>
-      &nbsp;({t(`group.role_${_.getOr('member', 'role', group)}`)})
-    </List>
+      &nbsp;
```

Comment:
question: this this to prevent a word wrap or to add whitespace? Can we uses CSS instead?

Reply:
@paulschreiber I think it was added to add the space (i don't remeber if I did this). I can use `theme.spacing()` to keep it consistent with the rest of the code